### PR TITLE
Supporting EditText and child realization for KTextInputLayout

### PR DESCRIPTION
**Pull request by v-rodionov**

### CODE REVIEW

**Comment by v-rodionov** [comment]
#164

File: kakao/src/main/kotlin/com/agoda/kakao/edit/KTextInputLayout.kt [hunk 1]
```diff
@@ -20,21 +21,21 @@ class KTextInputLayout : KBaseView<KTextInputLayout>, TextInputLayoutAssertions
     constructor(function: ViewBuilder.() -> Unit) : super(function) {
         edit = KEditText {
             isDescendantOfA(function)
-            isInstanceOf(TextInputEditText::class.java)
+            withMatcher(ViewMatchers.isAssignableFrom(EditText::class.java))
```
Comment (Unlimity) [inline kakao/src/main/kotlin/com/agoda/kakao/edit/KTextInputLayout.kt]: Can you please extract this matcher and make it available inside `ViewBuilder`?

Comment (v-rodionov) [reply kakao/src/main/kotlin/com/agoda/kakao/edit/KTextInputLayout.kt]: done

**Comment by Vacxe** [comment]
@v-rodionov Thanks for the contribution!

**Comment by Vacxe** [comment]
@v-rodionov small tips for you

You can add to PR
Close: #164 

And this issue will be closed automatically after PR will be merged

Cheers!

**Pull request merged**